src/qt/pivx/sendchangeaddressdialog.cpp

```diff
@@ -8,10 +8,15 @@
 #include "coincontrol.h"
 #include "qt/pivx/qtutils.h"
 
-SendChangeAddressDialog::SendChangeAddressDialog(QWidget *parent) :
+SendChangeAddressDialog::SendChangeAddressDialog(QWidget* parent, WalletModel* model) :
     QDialog(parent),
+    walletModel(model),
     ui(new Ui::SendChangeAddressDialog)
 {
+    if (!walletModel) {
+        LogPrintf("%s: ERROR: No wallet model set.\n", __func__);
```

Comment:
Two things about this: 

In QT we are not using `LogPrintf`, we are using `qDebug`.
`qDebug() << __func__ << ": No wallet model set.";`

And, if we really want to have this check, would rather throw a runtime exception. Constructors initialize objects or not, there is no half done initialization. Create a widget in an invalid state, because of a bad argument, without notifying the caller, shouldn't never happen.

Reply:
Yes. Makes sense to throw. Fixed.